### ISSUE TITLE
No accessibily statement

### ISSUE THREAD

**Issue by StephenPatterson-DSS** [dof-dss/architecture-docs#16]
**Describe the bug**
There is no accessibility statement on the website.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to the home page
2. View the footer

**Expected behaviour**
A link to an accessibility statement should be present.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
